static/app/components/workflowEngine/gridCell/userCell.tsx

```diff
@@ -0,0 +1,27 @@
+import styled from '@emotion/styled';
+
+import UserBadge from 'sentry/components/idBadge/userBadge';
+import {IconSentry} from 'sentry/icons';
+import type {AvatarUser} from 'sentry/types/user';
+
+export type UserCellProps = {
+  user: 'sentry' | AvatarUser;
+  className?: string;
+  disabled?: boolean;
+};
+
+export function UserCell({user, disabled = false, className}: UserCellProps) {
+  return (
+    <Wrapper disabled={disabled} className={className}>
+      {user === 'sentry' ? (
+        <IconSentry size="lg" />
+      ) : (
+        <UserBadge hideEmail hideName avatarSize={24} user={user} />
+      )}
+    </Wrapper>
+  );
```

Comment:
nit: is this a bit more reusable of a component outside of a cell? it kinda reminds me of an "avatar" component where we could have the UserBadge or a default sentry icon 🤔

Reply:
Yes but I'm not too sure where to fit it into the rest of the system right now. We already have quite a few `*Avatar` variants and this is an emerging pattern, so I don't want to stretch it too far before we understand what the generic constraints should be.